content/blog/2022-02-28-february-22-community-gems.md

```diff
@@ -0,0 +1,182 @@
+---
+title: February '22 Community Gems
+date: 2022-02-28
+description: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML community. This month:
+  comparing experiments, working with data, working with pipelines, and more.
+picture: 2022-01-31/jan-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/100132
+tags:
+  - Data Versioning
+  - DVC Remotes
+  - DVC API
+  - DVC Stages
+  - Community
+---
+
+### [Is there a proper way of deleting DVC tracked files from cloud storage?](https://discord.com/channels/485586884165107732/563406153334128681/927618225989111880)
+
+Thanks for the question @fireballpoint1!
+
+You can find the best way to delete files from your cloud storage in
+[our docs](https://dvc.org/doc/command-reference/gc#removing-data-in-remote-storage).
+Make sure you're super careful when deleting data from the cloud because it's an
+irreversible action. Here's an example of a deletion command that will clear out
+your workspace and the cloud storage.
```

Comment:
As you explain below, it's not clearing out your workspace, so I think this will cause confusion, especially since people already find `gc` confusing. It's clearing out everything from cloud storage _except_ what's references in your workspace.

Reply:
Then I'm confused too. 😅 I was looking through the docs, but I can't find how to clear out the workspace in addition to the cloud. Could you point me to it?

Reply:
Ah, that's because there isn't a way to clear out the workspace! The point of `gc` is to clear out old, unused data from your cache or remote storage. Presumably, whatever is currently in your workspace is data you still want to keep, so `gc` doesn't provide an option to get rid of that data.

Reply:
Thanks for the explanation! That actually makes a lot more sense now.